aws/biganimal-csp-preflight

```diff
@@ -266,6 +274,10 @@ account=$1
 region=$2
 
 [ -z "$account" ] && suggest "<account-id> is a required option!" alert && show_help && exit 1
+[ "$ha" = "true" ] && [ "$architecture" != "single" ] \
```

Comment:
I’m not really convinced by this choice because if we have both `-a` and `-x single` we are having two options that contradicts themselves and the final result can be a misleading one from a user perspective. I think that our `biganimal` CLI from a certain version on will simply stop using `-a` and start using `-x` and we can keep that incompatibility of those together here. Make sense to you or I missed some point in your suggestion?

Reply:
Actually my point is that we can set `architecture` in the beginning of the script and the following logic only respects the `architecture`. Then if we decide to remove `ha` we don't need to modify everywhere `ha` exists. I think a function like `set_architecture(ha, architecture)` is enough. Any thoughts?

Reply:
Sure, make sense to me!

Reply:
Thanks! It's LGTM.